e2e/node/resource.test.ts

```diff
@@ -137,12 +131,12 @@ describe("Authenticated end-to-end", () => {
   it("can create, delete, and differentiate between RDF and non-RDF Resources", async () => {
     const fileUrl = `${sessionResource}.txt`;
```

Comment:
Do we actually need this?

Or is the "overwriteFile" function name misleading?

I'm wondering if we could just overwrite the resource (even though it's a solidDataset) as a text file.